app/models/user.rb

```diff
@@ -37,6 +37,7 @@ class User < ActiveRecord::Base
   has_many :books, through: :purchases
   has_many :purchases
   has_and_belongs_to_many :projects, foreign_key: 'admin_user_id'
+  has_and_belongs_to_many :book_lists, foreign_key: 'admin_user_id'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Prefer `has_many :through` to `has_and_belongs_to_many`.